featuretools/tests/entityset_tests/test_last_time_index.py

```diff
@@ -180,8 +180,8 @@ def test_parent_no_time_index_missing(self, es, extra_session_df,
 
     def test_multiple_children(self, es, wishlist_df,
                                true_sessions_lti):
-        if ks and isinstance(es.dataframes[0], ks.DataFrame):
-            pytest.xfail('Cannot make index on a Koalas DataFrame')
+        if ps and isinstance(es.dataframes[0], ps.DataFrame):
```

Comment:
Can we update this to use the `if es.dataframe_type == Library.SPARK.value` approach that we are using elsewhere in this file? There are few locations in this file that could be updated.